packages/core/src/decorators/attachment/Attachment.ts

```diff
@@ -45,6 +47,16 @@ export class AttachmentData {
     }
   }
 
+  public getDataAsJson() {
+    if (this.base64 !== null && typeof this.base64 === 'string') {
```

Comment:
I would use just `this.base64` to also capture `base64` being undefined

```suggestion
if (this.base64 && typeof this.base64 === 'string') {
```